### PR TITLE
Docs: adds set up section with alertmanager, provisioning sections

### PR DESCRIPTION
**Pull request by brendamuir**
Renames fundamentals to Explore
Creates Set up section for Alerting config sections -- copied across basic alertmanager existing config
Added file provisioning docs (which will now live in alerting and not in administration).

I also haven't deleted the provisioning sections yet-as trying to check link etc. implications first. ultimately, it will be deleted from admin and link out to alerting docs.

### CODE REVIEW

**Comment by brendamuir** [comment]
Note: Set up section -- idea here is to also document the integration with oncall, expand on alertmanager, and any other config set up sections needed here.

File: docs/sources/alerting/set-up/_index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+weight: 107
+---
+
+# Set up Grafana Alerting
+
+Configure the features and integrations that you need to create and manage your alerts.
+
+- [Configure Alertmanager](https://grafana.com/docs/grafana/latest/alerting/set-up/configure-alertmanager/)
+- [Provision Grafana Alerting resources](https://grafana.com/docs/grafana/latest/alerting/set-up/provision-alerting-resources/)
+
```
Comment (armandgrillet) [inline docs/sources/alerting/set-up/_index.md:16]: Why is there a blank line here?

File: docs/sources/alerting/set-up/configure-alertmanager/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - set up
+  - configure
+  - Alertmanager
+title: Configure Alertmanager
+weight: 100
+---
+
+# Configure Alertmanager
+
+Configure Alertmanager for Grafana Alerting to group and manage alert rules, adding a layer of orchestration on top of the alerting engines.
```
Comment (armandgrillet) [inline docs/sources/alerting/set-up/configure-alertmanager/index.md:18]:
```suggestion
Configure Alertmanager from Grafana Alerting to group and manage alert rules, adding a layer of orchestration on top of your external alerting engine.
```

File: docs/sources/alerting/set-up/configure-alertmanager/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Add a new external Alertmanager
+
+1. In the Grafana menu, click the Alerting (bell) icon to open the Alerting page listing existing alerts.
+2. Click **Admin** and then scroll down to the External Alertmanager section.
+3. Click **Add Alertmanager** and a modal opens.
+4. Add the URL and the port for the external Alertmanager. You do not need to specify the path suffix, for example, `/api/v(1|2)/alerts`. Grafana automatically adds this.
```
Comment (armandgrillet) [inline docs/sources/alerting/set-up/configure-alertmanager/index.md:25]: @yuri-tceretian I remember our last customer call where this was more complicated than that, anything to add?

File: docs/sources/alerting/set-up/configure-alertmanager/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+The external URL is listed in the table with a pending status. Once Grafana verifies that the Alertmanager is discovered, the status changes to active. No requests are made to the external Alertmanager at this point; the verification signals that alerts are ready to be sent.
+
+### Edit an external Alertmanager
+
+1. Click the pen symbol to the right of the Alertmanager row in the table.
+2. When the edit modal opens, you can view all the URLs that were added.
+
+The edited URL will be pending until Grafana verifies it again.
+
+{{< figure max-width="40%" src="/static/img/docs/alerting/unified/ext-alertmanager-active.png" max-width="650px" caption="External Alertmanagers" >}}
```
Comment (armandgrillet) [inline docs/sources/alerting/set-up/configure-alertmanager/index.md:36]: Does this image exist?

File: docs/sources/alerting/set-up/provision-alerting-resources/index.md [hunk 5]
```diff
@@ -0,0 +1,726 @@
...
+1. Create a YAML or JSON configuration file.
+
+   Example configuration files can be found below.
+
+1. Add the file(s) to your GitOps workflow, so that they deploy alongside your Grafana instance(s).
+
+Here is an example of a configuration file for creating contact points.
+
+```yaml
+# config file version
```
Comment (armandgrillet) [inline docs/sources/alerting/set-up/provision-alerting-resources/index.md:170]: No uppercase in comment here but uppercase at "List of contact points", not consistent.

File: docs/sources/alerting/set-up/provision-alerting-resources/index.md [hunk 6]
```diff
@@ -0,0 +1,726 @@
...
+**Note:**
+
+Provisioning takes place during the initial set up of your Grafana system, but you can re-run it at any time using the [Grafana Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations).
+
+### Provision alert rules
+
+Create or delete alert rules in your Grafana instance(s).
+
+1. Create an alert rule in Grafana.
+1. Use the [Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations) to extract the alert rule.
```
Comment (armandgrillet) [inline docs/sources/alerting/set-up/provision-alerting-resources/index.md:59]: IMHO this is gonna be done by everyone thus we should provide more details regarding the call to the API here.

**Review by armandgrillet — APPROVED** [review]
A few nits but overall this is a great improvement.

File: docs/sources/alerting/set-up/_index.md (hunk 1 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/_index.md:16]: Very nitty. I kinda wanted to separate out the config steps and the integrations, but happy to put together until this gets beefed out and maybe gets two separate mini headings or so

File: docs/sources/alerting/set-up/provision-alerting-resources/index.md (hunk 6 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/provision-alerting-resources/index.md:59]: @JohnnyQQQQ can you help me here, please?

File: docs/sources/alerting/set-up/configure-alertmanager/index.md (hunk 3 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/configure-alertmanager/index.md:25]: This is next up after provisioning--i didn't write any of this - i jsut copied it as is from existing alertmanager docs. Yuri -- we can sit down and chat :)

File: docs/sources/alerting/set-up/configure-alertmanager/index.md (hunk 4 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/configure-alertmanager/index.md:36]: will check

File: docs/sources/alerting/set-up/provision-alerting-resources/index.md (hunk 5 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/provision-alerting-resources/index.md:170]: do you mean in the code?

File: docs/sources/alerting/set-up/_index.md (hunk 1 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/_index.md]: Updated:)

**Pull request merged**